internal/app/api/stripe.go

```diff
@@ -0,0 +1,104 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/formancehq/payments/internal/pkg/integration"
+
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/pkg/errors"
+
+	stripeConnector "github.com/formancehq/payments/internal/pkg/connectors/stripe"
+	"github.com/stripe/stripe-go/v72"
+	"github.com/stripe/stripe-go/v72/transfer"
+)
+
+type stripeTransferRequest struct {
+	Amount      int64             `json:"amount"`
+	Asset       string            `json:"asset"`
+	Destination string            `json:"destination"`
+	Metadata    map[string]string `json:"metadata"`
+
+	currency string
+}
+
+func (req *stripeTransferRequest) validate() error {
+	if req.Amount <= 0 {
+		return errors.New("amount must be greater than 0")
+	}
+
+	if req.Asset == "" {
+		return errors.New("asset is required")
+	}
+
+	if req.Asset != "USD/2" && req.Asset != "EUR/2" {
+		return errors.New("asset must be USD/2 or EUR/2")
+	}
+
+	req.currency = req.Asset[:3]
+
+	if req.Destination == "" {
+		return errors.New("destination is required")
+	}
+
+	return nil
+}
+
+func handleStripeTransfers(db *mongo.Database) http.HandlerFunc {
+	connectorStore := integration.NewMongoDBConnectorStore(db)
+	var cfg stripeConnector.Config
+
+	err := connectorStore.ReadConfig(context.Background(), stripeConnector.Name, &cfg)
```

Comment:
Reading config out of the request scope will "freeze" api key for the service instance.
Not allowing to update the api key if needed, without a restart of the service.
Right?